e2e/portalicious/pages/RegistrationsPage.ts

```diff
@@ -161,6 +240,185 @@ class RegistrationsPage extends BasePage {
     await this.table.tableRows.nth(row).click({ button: 'right' });
     await this.page.getByLabel(action).click();
   }
+
+  async clickAndSelectExportOption(option: string) {
+    await this.exportButton.click();
+    await this.page.getByRole('menuitem', { name: option }).click();
+  }
+
+  async assertExportButtonIsHidden() {
+    await expect(this.exportButton).toBeHidden();
+  }
+
+  async clickProceedToExport() {
+    await this.proceedButton.click();
+  }
+
+  async exportAndAssertData(
+    expectedColumns: string[],
+    assertionData: Record<string, unknown>,
+    registrationIndex: number,
+    filterContext?: string,
+    validateRowCount?: { condition: boolean; minRowCount: number },
+  ) {
+    const [download] = await Promise.all([
+      this.page.waitForEvent('download'),
+      this.clickProceedToExport(),
+    ]);
+
+    const downloadDir = './downloads';
```

Comment:
I guess this makes the download-location relative to "wherever the script has been run from", instead of a single/predictable location (which can/should be ignored by Git.)

So, by using:
```suggestion
    const downloadDir = path.join(__dirname, '../../downloads');
```

I think it would _always_ use the location: `<git-repo-root>/e2e/downloads`

(I haven't verified locally, so you might want to check first. :) )

Reply:
Works both locally and on pipeline